schema.sql

```diff
@@ -8,3 +8,9 @@ CREATE TABLE animals(
    neutered         BOOLEAN,
    weight_kg       DECIMAL
 );
+
+ALTER TABLE animals ADD species text NULL;
+
+BEGIN
+ALTER TABLE animals RENAME COLUMN species TO unspecified;
+ROLLBACK
```

Comment:
This is a query, therefore it should be in the `queries.sql` file. Everything that is a query that helps to analyze data should be saved in the queries.sql file. 

- Kindly make this correction.